README.md

```diff
@@ -1,6 +1,4 @@
-[![Slack Status](http://sdlslack.herokuapp.com/badge.svg)](http://slack.smartdevicelink.com)
-
-[![codecov.io](https://codecov.io/github/smartdevicelink/sdl_core/coverage.svg?branch=develop)](https://codecov.io/github/smartdevicelink/sdl_core?branch=develop)
+[![Slack Status](http://sdlslack.herokuapp.com/badge.svg)](http://slack.smartdevicelink.com)  [![codecov.io](https://codecov.io/github/smartdevicelink/sdl_core/coverage.svg?branch=develop)](https://codecov.io/github/smartdevicelink/sdl_core?branch=develop)  [![Build Status](http://opensdl-jenkins.luxoft.com:8080/view/All/job/OpenSDL_PR/badge/icon)](http://opensdl-jenkins.luxoft.com:8080/view/All/job/OpenSDL_PR/)
```

Comment:
@vkushnirenko-luxoft please split 3 tags by 3 lines in raw code. Markdown will ignore one '\n\ and view won't be corrupted.

Reply:
@vkushnirenko-luxoft why do we need label for PR job in develop README? Each time if ANY pull request is READ lable gona be red? This lable in develop should show state of current develop.